src/main/java/frc/robot/Constants.java

```diff
@@ -261,16 +261,16 @@
      * Constants for Inner Magazine PID
```

Comment:
U can delete the Shooter Roller PID constants.  Also did u update the Shooter PID values??